src/main/java/com/spotify/docker/Utils.java

```diff
@@ -32,7 +32,11 @@
 
 public class Utils {
 
-  public static String[] parseImageName(String imageName) {
+  public static String[] parseImageName(String imageName) throws MojoExecutionException {
+    if (imageName == null) {
```

Comment:
Use `isNullOrEmpty` instead of just checking for null (empty and null strings should generally be treated the same).